src/main/java/com/ratifire/devrate/repository/interview/InterviewRequestRepository.java

```diff
@@ -50,4 +52,6 @@ List<InterviewRequest> findMatchedInterviewers(@Param("request") InterviewReques
       List<User> ignoreList);
 
   List<InterviewRequest> findByActiveTrueAndExpiredAtBefore(ZonedDateTime currentDateTime);
+
+  Optional<InterviewRequest> findByUserAndRole(User user, InterviewRequestRole role);
```

Comment:
What if try to rename method findByUserIdAndRole(long userId, InterviewRequestRole role) ?